service.datadog.yaml

```diff
@@ -0,0 +1,13 @@
+---
+schema-version: v2
+dd-service: datadog-agent
+
+extensions:
+  datadoghq.com/sdp:
+    workday_team: "Agent Platform"
+    conductor:
+      slack: "datadog-agent-pipelines"
+      schedule: "0 3 3 * * 1-5"
+      env:
+        - name: "Nightly builds"
+          branch: "chouquette/conductor"
```

Comment:
Presumably this is intentional for now and it should be `main` once the feature is stabilized?

Reply:
Indeed maybe it makes more sense to merge without this, and add it once the service is known to DDR